mobile/src/main/res/values/themes.xml

```diff
@@ -34,6 +34,29 @@
         <item name="colorPrimaryDark">@color/blue_grey_900</item>
     </style>
 
+    <style name="openHAB.Amoled" parent="openHAB.DayNight">
+        <item name="chartTheme">black</item>
+        <item name="android:windowBackground">@color/black</item>
```

Comment:
I'm not sure if changing the charts and background from dark grey to black is enough. I'll ask in the forum for feedback.

Reply:
I think you'll want to at least change the card background for frames. I think it's `#111` by default.

Reply:
I'm not sure what you mean. The app doesn't use cards for frames, only basic ui does this.

Reply:
Never mind, I thought we are using cards for the frames.